UpdateManager.js

```diff
@@ -0,0 +1,30 @@
+const axios = require('axios');
+
+class UpdateManager {
+    constructor({ interval }) {
+        this.interval = interval || 1000 * 60 * 60;
+    }
```

Comment:
Is there a good way to turn this on by default? Like maybe call start in the constructor (why else would they instantiate the class)? Also maybe by creating and holding an instance of the update manager in the `NetworthManager` class with flags to control if it's used?

Again, this is likely a singleton class.

Reply:
yea we can do that lmk the specifics how you would think is the best way of doing this